pubspec.yaml

```diff
@@ -64,6 +64,7 @@ dependencies:
   wakelock_plus: ^1.2.8
   zulip_plugin:
     path: ./packages/zulip_plugin
+  permission_handler: ^11.3.1
   # Keep list sorted when adding dependencies; it helps prevent merge conflicts.
```

Comment:
Please follow the request in the comment. Also, we generally use a separate commit to add dependencies; see the [project history](https://github.com/zulip/zulip-mobile/blob/main/docs/howto/git.md#2-the-secret-to-using-git-log--p) for examples.

Reply:
changed the order maintaining the list sorted